docs/pages/versions/unversioned/sdk/audio.mdx

```diff
@@ -2,35 +2,71 @@
 title: Audio
```

Comment:
One thing that I found annoying in Video is the lack of the package name here. Of course we'll remove it once `expo-av` docs are removed.
```suggestion
title: Audio (expo-audio)
```